app/src/main/res/layout/notification_dialog.xml

```diff
@@ -41,20 +41,20 @@
         android:inputType="textPersonName"
         android:text="Title"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/closeButton" />
+        app:layout_constraintTop_toBottomOf="@+id/btnClose" />
 
     <Button
-        android:id="@+id/btnconfirm"
+        android:id="@+id/btnConfirm"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="16dp"
         android:layout_marginTop="20dp"
         android:text="Confirm"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/notificationText" />
+        app:layout_constraintTop_toBottomOf="@+id/txtNotification" />
 
     <Button
-        android:id="@+id/closeButton"
+        android:id="@+id/btnClose"
```

Comment:
Suggestion 3: Here is where I changed the ID names of some of the objects in the notification_dialog XML file.